lib/db.js

```diff
@@ -120,9 +120,10 @@ api.init = function (app, callback) {
 };
 
 api.getSnapshots = function getSnapshots (opts, callback) {
+  var searchOpts = _.merge({'status': 'SUCCESSFUL'}, opts);
```

Comment:
in JavaScript, object keys don't need to be quoted. Just have:
`{status: 'SUCCESSFUL'}`